homeassistant/components/synology_dsm/const.py

```diff
@@ -227,15 +228,15 @@
     f"{SynoStorage.API_KEY}:disk_smart_status": {
         ENTITY_NAME: "Status (Smart)",
         ENTITY_UNIT: None,
-        ENTITY_ICON: "mdi:checkbox-marked-circle-outline",
-        ENTITY_CLASS: None,
+        ENTITY_ICON: None,
+        ENTITY_CLASS: DEVICE_CLASS_SAFETY,
```

Comment:
I suggest we revert these changes for now since it's not correct for these regular sensors.

Reply:
done